sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuDeviceManager.scala

```diff
@@ -206,6 +206,10 @@ object GpuDeviceManager extends Logging {
         init = init | RmmAllocationMode.POOL
         features += "POOLED"
       }
+      if (conf.isArenaMemEnabled) {
+        init = init | RmmAllocationMode.ARENA
+        features += "ARENA"
```

Comment:
This should check for asking for POOLED and ARENA at the same time, if such a configuration setup by the user is possible, and throw an exception with a useful error message indicating what the config conflict is so the user knows how to fix it.  Otherwise the error will come from cudf and not tie back to plugin configs that the user will be familiar with.

Reply:
Done.